CMakeLists.txt

```diff
@@ -158,13 +158,24 @@ if (XSDK_ENABLE_Fortran)
    enable_language(Fortran)
 endif()
 
-# Suppress warnings about MACOSX_RPATH
-set(CMAKE_MACOSX_RPATH OFF CACHE BOOL "")
+set(CMAKE_MACOSX_RPATH ON CACHE BOOL "")
+
+if (APPLE)
+   # Set policy CMP0068 to NEW so that install_name will include RPATH settings
+   if(POLICY CMP0068)
+       cmake_policy(SET CMP0068 NEW)
+   endif()
+
+   set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
```

Comment:
If we do this, do we need to set `BUILD_WITH_INSTALL_RPATH` later on?
And, again, if we're doing this, shouldn't we also set `CMAKE_INSTALL_RPATH` and `CMAKE_INSTALL_NAME_DIR` instead?

Reply:
followed your suggestion in https://github.com/mfem/mfem/pull/4583/commits/ddfc59f3c321b7ccbed0e9fcffb6f0983ab49739

Reply:
Check Slack.